notebooks/Watermark photo attachments with Exif data.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Watermark photo attachments with Exif data\n",
+    "\n",
+    "This notebook demonstrates how to watermark photo attachments with latitude and longitude from Exif GPS info using the Pillow Library.\n",
+    "\n",
+    "To use this notebook, you need to update variables at the top of the script, including:\n",
+    "\n",
+    "1. feature_layer_item_id to the item id of the feature layer to watermark attachments for and feature_layer_id to specify the specific layer\n",
+    "2. attachments_query to filter what features to watermark their attachments\n",
+    "\n",
+    "This notebook assumes that:\n",
+    "\n",
+    "- You are the owner of the webmap\n",
+    "- You are using Enterprise 10.7+ or ArcGIS Online\n",
+    "\n",
+    "For more information on working with map areas see:\n",
+    "\n",
+    "- ArcGIS Python API Layer Attachments - https://developers.arcgis.com/python/guide/using-attachments-with-feature-layers/\n",
+    "- Pillow Library - https://pillow.readthedocs.io/en/stable/index.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import shutil\n",
+    "import tempfile\n",
+    "from arcgis.gis import GIS\n",
+    "from datetime import datetime as dt\n",
+    "from PIL import ExifTags\n",
+    "from PIL import Image\n",
+    "from PIL import ImageDraw\n",
+    "from PIL import ImageOps\n",
+    "\n",
+    "\n",
+    "# Using built-in user\n",
+    "# For information on different authentication schemes see\n",
+    "# https://developers.arcgis.com/python/guide/working-with-different-authentication-schemes/\n",
+    "# and for protecting your credentials see\n",
+    "# https://developers.arcgis.com/python/guide/working-with-different-authentication-schemes/#protecting-your-credentials\n",
+    "gis = GIS(\"home\")\n",
```

Comment:
Are users planning to use this in the hosted arcgis notebooks only?

Reply:
Not sure, guessing users may use this hosted or local.  The doc linked on line 46 should cover all the different options that they might need